compiler/cpp/src/generate/t_rs_generator.cc

```diff
@@ -450,7 +450,7 @@ string t_rs_generator::render_rs_type(t_type* type) {
     case t_base_type::TYPE_I64:
       return "i64";
     case t_base_type::TYPE_DOUBLE:
-      return "f64";
+      return "OrderedFloat<f64>";
```

Comment:
Hmm, maybe so. I wanted to use a std collection just because they tend to be the best supported.